### PR TITLE
[Fixes: #10471] Parse markdown in chat subheader.

### PR DESCRIPTION
**Pull request by cammellos**
This commit adds parsing of markdown in the chat subheader.

Because of the added complexity performance are impacted but I have
also noticed that on each loading of the chat screen we calculate
alias & identicon through status-go, so that has been changed so that
they are returned from status-go.

Overall performance is now roughly identical, tested loading 150
one-to-one chats.

Another issue that I've spotted is that some `subs` are unnecessarely
recalculated (`active-chats`), when unrelated fields changes. I will
address this in a separate PR that should improve performance.

status: ready (I need to test on a real device, but ready for review)

### CODE REVIEW

File: src/status_im/chat/db.cljs [hunk 1]
```diff
@@ -29,15 +32,25 @@
         (assoc :inviter-name inviter-name)
         :always
         (assoc :chat-name (group-chat-name chat))))
-    (let [{contact-name :name :as contact}
+    (let [photo (if (seq identicon)
```
Comment (flexsurfer) [inline src/status_im/chat/db.cljs:35]: why seq ?

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/chat/db.cljs (hunk 1 above)
Comment (cammellos) [reply src/status_im/chat/db.cljs:35]: We could use `blank?` but we just want to check if it's not `""` or `nil`(i.e empty)

**Comment by Serhy** [comment]
@cammellos 
Tested on Android 8.1 and iOS 13

**Issue 1** Both Android and iOS Empty space on chat preview when URL sent in the message.
Screenshot 2020-05-22 at 17 19 22

**Issue 2** On iOS only. The code block (the one which is tripple-backquotes): makes some gap between chats. 
To reproduce: send in chat the ` ``` codeblock ``` `  message and check message preview on Chats screen:

Screenshot 2020-05-22 at 17 19 34

**Comment by cammellos** [comment]
@Serhy I believe I have addressed both issues, I don't own an IOS device so could not test issue 2 specifically, if you would not mind testing them. Thanks!

**Comment by cammellos** [comment]
(in the new build, building now)

**Comment by Serhy** [comment]
Right, @cammellos that fixed both issues, can't reproduce with iOS and Android anymore.

**Pull request merged**